pkg/controllers/namespace/namespace_controller.go

```diff
@@ -120,12 +120,12 @@ func (r *NamespaceReconciler) createDefaultSubnetSet(ns string, defaultPodAccess
 					},
 				},
 			}
-			if name == types.DefaultVMSubnetSet {
```

Comment:
We still need this logic to set default VM/Pod access mode for SubnetSet CR, Private for VM default SubnetSet, PrivateTGW for Pod default SubnetSet, right? @lxiaopei

Reply:
Done, see also `Test Done`.